src/pages/user/edit-post.tsx

```diff
@@ -0,0 +1,9 @@
+import { EditPostCard } from "../../features/user/components/edit-post-card"
+
+export const EditPost = () => {
+  return(
+     <>
+          <EditPostCard />  
```

Comment:
tu tez, useless component, doslownie nic nie robi

Reply:
Chciałem zrobić po prostu Page oddzielnie od Future, i Pages to po prostu takie kontenery dla ficzerów.